example.proto

```diff
@@ -41,11 +41,13 @@ message Message {
 
 message Sub {
     enum Nested {
-        FOO = 0;
-        BAR = 1;
+        NONE = 0 [(huma.exclude) = true];
+        FOO = 1;
+        BAR = 2;
     }
 
     Nested testEnum = 1 [(huma.public) = true];
+    Nested test_enum2 = 2 [(huma.public) = true];
```

Comment:
Is there a reason for the change of `camelCase` to `snake_case` between the two tests?
Also, these two values look identical; should we expect the test results to differ?

Reply:
The casing was originally to ensure our casing code is working properly regardless of how people might do it in proto. Only needed one so I used the "normal" way of casing for the new value. The existence of this field in the test message is just to test unexported enum values to make sure they don't get serialized.

Reply:
Can we make this `camelCastTest` and `snake_case_test` so that intent is clear?